core/src/test/java/org/elasticsearch/common/logging/log4j/LoggingConfigurationTests.java

```diff
@@ -151,6 +154,28 @@ public void testResolveConfigInvalidFilename() throws Exception {
         assertThat(logSettings.get("yml"), Matchers.nullValue());
     }
 
+    // tests that config file is not read when we call LogConfigurator.configure(Settings, false)
+    @Test
+    public void testConfigNotRead() throws Exception {
+        Path tmpDir = createTempDir();
+        Path loggingConf = tmpDir.resolve(loggingConfiguration("yaml"));
+        Files.write(loggingConf, "logger.test: INFO, console\n".getBytes(StandardCharsets.UTF_8));
+        Files.write(loggingConf, "appender.console.type: console\n".getBytes(StandardCharsets.UTF_8), StandardOpenOption.APPEND);
```

Comment:
I wonder if something like:
```java
Files.write(loggingConf, Arrays.asList(
  "logger.test: INFO, console",
  "appender.console.type: console"));
```
would be a bit easier to read.

That Files.write method is documented as using UTF-8 by default.